src/modules/launcher/Wox.Infrastructure/Image/WindowsThumbnailProvider.cs

```diff
@@ -4,17 +4,21 @@
 
 using System;
 using System.IO.Abstractions;
+using System.Reflection;
 using System.Runtime.InteropServices;
 using System.Windows;
 using System.Windows.Interop;
 using System.Windows.Media.Imaging;
+using Microsoft.PowerToys.Settings.UI.Library.Utilities;
```

Comment:
Why do we need it?

Reply:
@htcfreek 

It was from me "auto-completing" in the wrong logger. Really disliking the Visual Studio behavior of automatically adding "using" clauses to my code when I wrongly auto complete some name 🤷 

Thanks for spotting it. 😄

Reply:
I hate it too. If you find a setting please notify me. 😅

Reply:
@htcfreek , Guess I'm trying turning off that "Add missing using directives on paste"
![image](https://user-images.githubusercontent.com/26118718/216953371-6faa78e0-3d28-4668-95e8-4f0c2eed86e7.png)